apis/alertcondition/v1alpha1/register.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+// Package type metadata.
+const (
+	Group   = "alertcondition.provider-newrelic.crossplane.io"
```

Comment:
I wonder if we might move the resources added in this PR into a single group? As is, I believe you will have something like `alertcondition.alertcondition.provider-newrelic.crossplane.io/v1alpha1`, which is duplicative. What do you think?

Reply:
Yes - at one point I grouped `policy` and `nrqlcondition` under `alerts` - I could go either way.
As it is now, the APIs are
`apiVersion: alertpolicy.provider-newrelic.crossplane.io/v1alpha1`
` apiVersion: alertnrqlcondition.provider-newrelic.crossplane.io/v1alpha1`
which I don't think is so bad

I am planning on adding more resources very soon and was thinking that might affect the structure as there are separate APIs.